jobs/mozillians_magic.rb

```diff
@@ -42,19 +41,22 @@ def query_mozillians(params = {})
       http.request(request)
     end
 
-    def query_mozillians_group(group)
-      response = query_mozillians({ group: group })
-
-      if response.code.to_i == 200
-        res = JSON.parse(response.body)
-        count = res['count']
+    def iterate_mozillians_groups(groups)
+      remove_from_group('mozillians_nda')
+      remove_from_group('mozillians_iam_project')
+      groups.each do |group|
+        name = group['name']
+        add_to_group('mozillians_nda') if name == 'nda'
+        add_to_group('mozillians_iam_project') if name == 'iam-project'
+      end
+    end
 
-        if count.to_i == 1
-          add_to_group("mozillians_#{group}")
-        else
-          remove_from_group("mozillians_#{group}")
-        end
+    def query_mozillians_profile(_url)
+      response = query_mozillians({}, _url)
```

Comment:
This way no params are sent at all, rather than `?email=` (or the POST equivalent)